src/applications/gi/components/profile/Programs.jsx

```diff
@@ -53,6 +53,12 @@ export class Programs extends React.Component {
         text: 'Military Tuition Assistance (TA)',
         link: false
       },
+
+      priorityEnrollment: {
```

Comment:
new attributes are returned to the institution end point `priority_enrollment` and `independent_study`. The keys are used directly here and [here](https://github.com/department-of-veterans-affairs/vets-website/blob/1178155e3340b103c17f09734d9302fa6dd0cabe/src/applications/gi/components/profile/AdditionalInformation.jsx#L74).